### ISSUE TITLE
Falcon Loss Not Decreasing During Training

### ISSUE THREAD

**Issue by LamOne1** [Lightning-AI/lit-gpt#125]
I'm using pretrain code with falcon 7B. I've noticed that the loss didn't change for 400 iterations. 

```
iter 1: loss 11.0666, time: 13381.00ms, speed: 306 toks/s/device
....
iter 400: loss 11.0666, time: 19090.34ms, speed: 214 toks/s/device
```

**Comment by carmocca** [comment]
Can you share exactly what script you ran?

**Comment by LamOne1** [comment]
I was using redpajama pretrain script as it is.
However I prepared my data with PackedDataset type np.uint16 instead of np.int32, could be the problem?

**Comment by carmocca** [comment]
I don't think so. But you might need to tweak hyperparameters. This is the dark art of machine learning :wink:
